RecoTracker/TrackProducer/src/KfTrackProducerBase.cc

```diff
@@ -121,15 +121,20 @@ void KfTrackProducerBase::putInEvt(edm::Event& evt,
 						 innertsos.curvilinearError(), innerId,
     						 seedDir, theTraj->seedRef()));
 
+    assert(!useSplitting);
```

Comment:
Some comment would be nice (either commented out or a part of the assert/Exception message)